lite2/client.go

```diff
@@ -89,7 +89,16 @@ func SkippingVerification(trustLevel tmmath.Fraction) Option {
 // current primary is unavailable.
 func Witnesses(providers []provider.Provider) Option {
 	return func(c *Client) {
-		c.witnesses = providers
+		validWitnesses := make([]provider.Provider, 0)
```

Comment:
nit: more idiomatic to say `var validWitness []provider.Provider` if you don't know the size of the slice you want to allocate